gunicorn_thrift/gevent_worker.py

```diff
@@ -45,6 +45,10 @@ def handle(self, listener, client, addr):
 
             try:
                 while True:
+                    self.nr += 1
+                    if self.alive and self.nr >= self.max_requests:
+                        self.log.info("Autorestarting worker after current process.")
+                        self.alive = False                    
```

Comment:
Worker will exit  until graceful_timeout reached.

I don't know how to stop worker immediately when it reaches max_requests. 
Any advices are appreciated.